vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -149,8 +181,169 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
                 - My first page
              Second title:
                 - My second page
+            selector:
+                type: navbar
+
         ```
     === "Result"
-        [![CustomNavigation]][CustomNavigation]
+        [![NavigationIcons]][NavigationIcons]
 
-    [CustomNavigation]: ../../assets/user_guides/navigation/custom_navigation.png
+    [NavigationIcons]: ../../assets/user_guides/navigation/navigation_icons.png
+
+
+## Customizing Selectors
+
+### Accordion
+To customize default accordion provide a `dict` mapping of the page group title and a list of page IDs to the `pages` argument of the [`Accordion`][vizro.models.Accordion] model. 
+
+   ```
+    Navigation(
```

Comment:
Shall we add the model import to all these code snippets, so it's clear what are Vizro models?
```py
    import vizro.models as vm

    vm.Navigation(
        selector=vm.Accordion(pages={"Title": ["My first page", "My second page"]})
    )
   ```

The same applies to other snippets